settings.json

```diff
@@ -77,5 +77,12 @@
       "appId": "foo",
       "secret": "bar"
     }
-  }
+  },
+  "galaxy.meteor.com": {
+       "env": {
+          "MONGO_URL": "mongodb://habryka:6fpfghyLnVIP@ds139042-a0.mlab.com:39042,ds139042-a1.mlab.com:39042/lesswrong2?replicaSet=rs-ds139042",
```

Comment:
wow your database is public lol :D 
mongodb://habryka:6fpfghyLnVIP@ds139042-a0.mlab.com:39042,ds139042-a1.mlab.com:39042/lesswrong2?replicaSet=rs-ds139042